widgetssdk/src/main/java/com/glia/widgets/call/CallConfiguration.kt

```diff
@@ -1,33 +1,33 @@
 package com.glia.widgets.call
 
 import com.glia.androidsdk.Engagement
-import com.glia.widgets.core.configuration.GliaSdkConfiguration
+import com.glia.widgets.core.configuration.EngagementConfiguration
 import com.glia.widgets.helper.Utils
 
-internal class Configuration private constructor(builder: Builder) {
+internal class CallConfiguration private constructor(builder: Builder) {
```

Comment:
I have double checked the usages. Corrected the name for one variable.
The answer is - yes, the `CallConfiguration` class is used for calls only.
So, instead of 'Configuration' name that tells literally nothing it makes sense to have a more specific `CallConfiguration` name.

Reply:
Looks good now.